packages/xml-views-completion/api.d.ts

```diff
@@ -122,3 +129,38 @@ export interface UI5NamespacesInXMLAttributeValueCompletion
   extends BaseXMLViewCompletion<XMLAttribute, UI5Namespace> {
   type: "UI5NamespacesInXMLAttributeValue";
 }
+
+export type LiteralXMLViewCompletionBaseType = boolean | string | number;
```

Comment:
why can't we simply represent the literal as a plain string ?
Why do we need to distinguish between boolean/string/number here?

Reply:
It's all strings at the end because they are an xml attribute values which are written to a text document, but string values for example should be escaped while boolean and numbers don't.
I think it's more accurate to use the original type. Technically they are converted to string afterwards but I don't think the conversion should be here. (Like we keep the original value for default values in properties.)